ansible/roles/test/tasks/link_flap/link_flap_helper.yml

```diff
@@ -38,11 +38,11 @@
       assert: { that: "'{{ ansible_interface_facts[item]['active'] }}' == 'True'" }
       with_items: "{{ minigraph_ports }}"
       when: item != "{{ intfs_to_exclude }}"
+      connection: local
```

Comment:
I don't think to add `connection: local` is needed here.
Ansible will determine whether a command needs to be executed locally or not. It would be better to remove the conditions here.

Reply:
Oops, I merged already.

But making it explicit should be good too?

If you insist, I can remove it when I am making the interface_facts.py change.

Cheers,
Ying

Reply:
please remove